client/src/components/relatedProducts/card.jsx

```diff
@@ -106,12 +114,19 @@ const Card = (props) => {
 
   const handleClick = (event) => {
     console.log(props.info)
-    props.handleChangeProduct(props.info)
+    clearInterval(intervalID)
+    props.handleChangeProduct(props.info, props.style)
   }
 
   return (
     <Carta >
-      <RelatedBtn>{props.button}</RelatedBtn>
+      <RelatedBtn  onClick={()=>{
+        console.log(`clicked ${props.btnStyle} button`)
+        props.btnStyle === 'x' ? props.handleXClick() : props.setModalOpen(true)
+        }}>
+        {/* {props.button} */}
+        {props.btnStyle === 'star' ? <FontAwesomeIcon  style={second} icon={faStar} />: <FontAwesomeIcon  style={second} icon={faCircleXmark} />}
+        </RelatedBtn>
```

Comment:
You left the console log in on line 124.

Reply:
Im keeping it for know, it only logs to console when clicked